sktime/classification/feature_based/tests/test_random_interval_classifier.py

```diff
@@ -0,0 +1,220 @@
+    )
+    ric.fit(X_train, y_train)
+
+    # assert probabilities are the same
+    probas = ric.predict_proba(X_test.iloc[indices])
+    testing.assert_array_almost_equal(
+        probas, random_interval_classifier_unit_test_probas, decimal=2
+    )
+
+
+def test_random_interval_classifier_on_basic_motions():
+    """Test of RandomIntervalClassifier on basic motions."""
+    # load basic motions data
+    X_train, y_train = load_basic_motions(split="train", return_X_y=True)
+    X_test, y_test = load_basic_motions(split="test", return_X_y=True)
+    indices = np.random.RandomState(4).choice(len(y_train), 10, replace=False)
+
+    # train random interval classifier
+    ric = RandomIntervalClassifier(
+        random_state=0,
+        n_intervals=5,
+        interval_transformers=SummaryTransformer(
+            summary_function=("mean", "std", "min", "max"),
+            quantiles=(0.25, 0.5, 0.75),
+        ),
+        estimator=RandomForestClassifier(n_estimators=10),
+    )
+    ric.fit(X_train.iloc[indices], y_train[indices])
+
+    # assert probabilities are the same
+    probas = ric.predict_proba(X_test.iloc[indices])
+    testing.assert_array_almost_equal(
+        probas, random_interval_classifier_basic_motions_probas, decimal=2
+    )
+
+
+random_interval_classifier_unit_test_probas = np.array(
+    [
+        [
+            0.0,
+            1.0,
+        ],
+        [
+            1.0,
+            0.0,
+        ],
+        [
+            0.0,
+            1.0,
+        ],
+        [
+            1.0,
+            0.0,
+        ],
+        [
+            1.0,
+            0.0,
+        ],
+        [
+            1.0,
+            0.0,
+        ],
+        [
+            0.9,
+            0.1,
+        ],
+        [
+            0.2,
+            0.8,
+        ],
+        [
+            0.9,
+            0.1,
+        ],
+        [
+            0.9,
+            0.1,
+        ],
+    ]
+)
+random_interval_classifier_basic_motions_probas = np.array(
+    [
+        [
+            0.0,
+            0.0,
+            0.2,
+            0.8,
+        ],
+        [
+            0.2,
+            0.3,
+            0.1,
+            0.4,
+        ],
+        [
+            0.0,
+            0.0,
+            0.8,
+            0.2,
+        ],
+        [
+            0.2,
+            0.6,
+            0.0,
+            0.2,
+        ],
+        [
+            0.0,
+            0.0,
+            0.2,
+            0.8,
+        ],
+        [
+            0.0,
+            0.1,
+            0.5,
+            0.4,
+        ],
+        [
+            0.3,
+            0.2,
+            0.1,
+            0.4,
+        ],
+        [
+            0.0,
+            0.0,
+            0.9,
+            0.1,
+        ],
+        [
+            0.0,
+            0.9,
+            0.0,
+            0.1,
+        ],
+        [
+            0.2,
+            0.8,
+            0.0,
+            0.0,
+        ],
+    ]
+)
+
+
+# def print_array(array):
```

Comment:
please remove commented out code

Reply:
That is the code for generating the results we compare against, left in with the reasoning that it would be very annoying to have to re-implement it every time the results have to be redone after a legitimate change.

I'm against removing it completely, but we could consider formalising it a bit more, i.e. relocating the print function to utils and cleaning up the main portion a bit.